liquibase-standard/src/main/java/liquibase/command/core/UpdateCountCommandStep.java

```diff
@@ -115,11 +111,7 @@ protected void customMdcLogging(CommandScope commandScope) {
     }
 
     @Override
-    public void postUpdateLog(int rowsAffected) {
-        if (rowsAffected > -1) {
-            Scope.getCurrentScope().getUI().sendMessage(String.format(coreBundle.getString("update.count.successful.with.row.count"), rowsAffected));
```

Comment:
Update-count has been replaced to Update has been successful as noticed by @rberezen

Reply:
@rberezen fixed!